client/my-sites/plugins/plugin-sections/custom.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionNav from 'components/section-nav';
+import { sanitizeSectionContent } from 'lib/plugins/utils';
+
+const PluginSectionsCustom = ( { plugin, translate } ) => {
+	if ( ! plugin.description.length ) {
```

Comment:
I think it would be good to do this check on the sanitized content. In some cases we might have a description but sanitize it away.